pkg/tlsf/.gitattributes

```diff
@@ -1,2 +0,0 @@
-# Ignore whitespace errors inside these patch files by regarding them as binary.
-patch.txt binary
```

Comment:
Yes! Diffs of diffs are already hard to look at as is. Disabling diffs (which - in short - declaring it a binary file does) altogether makes it even harder to spot the differences to a package in a commit.